packages/ai-ollama/src/node/ollama-language-model.ts

```diff
@@ -203,18 +203,18 @@ export class OllamaModel implements LanguageModel {
             if (!props) {
                 return undefined;
             }
-            const result: Record<string, { type: string, description: string }> = {};
-            for (const key in props) {
-                if (Object.prototype.hasOwnProperty.call(props, key)) {
-                    const type = props[key].type;
-                    if (!type) {
-                        // Todo: Should handle anyOf, but this is not supported by the Ollama type yet
-                    } else {
-                        result[key] = {
-                            type: type,
-                            description: key
-                        };
+
+            const result: Record<string, { type: string, description: string, enum?: string[] }> = {};
+            for (const [key, prop] of Object.entries(props)) {
+                const type = prop.type;
+                if (type) {
+                    const description = typeof prop.description == 'string' ? prop.description : '';
+                    result[key] = {
+                        type: type,
+                        description: description
```

Comment:
Do we need this type check? I would have expected
```suggestion
                    const description = typeof prop.description == 'string' ? prop.description : '';
                    result[key] = {
                        type: type,
                        description: prop.description ?? key
```